site/src/app.css

```
@@ -2,7 +2,10 @@
   --night: #061e25;
   --blue: cornflowerblue;
   --text-color: rgb(208, 208, 208);
+  --text: var(--text-color);
   --max-main-width: 55em;
+  --main-max-width: var(--max-main-width);
+  color-scheme: dark;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)